azure-pipelines.yml

```diff
@@ -10,6 +10,6 @@ pr:
 - master
 
 jobs:
-  - template: ./.azure-pipelines/azure-pipelines-linux.yml
+  # - template: ./.azure-pipelines/azure-pipelines-linux.yml
   - template: ./.azure-pipelines/azure-pipelines-osx.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
+  # - template: ./.azure-pipelines/azure-pipelines-win.yml
```

Comment:
Why are these commented?

Reply:
I commented them because the recipe I intend to upload on conda-forge is for Mac-only at the moment. I am planning on adding Linux, but I was having trouble in creating an environment suitable for the compilation of the GUI

Reply:
It shouldn't be necessary to change this file. The `skip` in the recipe should be sufficient. Could you please revert this?

Reply:
👍🏻 Reverted